src/model/local.go

```diff
@@ -2,22 +2,26 @@ package model
 
 import (
 	"fmt"
-	"keboola-as-code/src/utils"
 )
 
-func LoadLocalState(projectDir string, metadataDir string) (*State, *utils.Error) {
+func LoadLocalState(projectDir string, metadataDir string) (*State, *PathsState, error) {
+	// Create structures
 	state := NewState()
+	paths, err := NewPathsState(projectDir)
```

Comment:
`paths` uchovávajú, ktoré lokálne súbory su tracked -> maju pre `keboola-as-code` vyznam a ktore nie.
Staci napr. ze uzivatel si vytvori nejaky config rucne, ale nebude v manifeste -> `untracked` (to ci by sme mohli v buducnosti take nieco povolit, teraz neriesim).